torch/csrc/jit/serialization/source_range_serialization.cpp

```diff
@@ -10,18 +11,26 @@ namespace jit {
 class SourceRangeSerializer {
  public:
   // Serialize SourceRange as Tuple[SourceType, int, int]
-  // where SourceType = Tuple[str, Optional[str], int, List[int]],
+  // where SourceType = Tuple[int, int, int, List[int]],
+  // The first 2 ints are positions into the vector returned by textSaved
+  // after all the Ranges are processed. textSaved() returns a vector of str
   // the serialized form of Source
   c10::IValue serialize(const SourceRange& sr);
 
+  const std::vector<c10::IValue>& textsSaved() {
```

Comment:
I think naming convention for methods in this file is lower case words connected by underscores.